### ISSUE TITLE
Error: Undefined symbol: Caster in pycaster.py

### ISSUE THREAD

**Issue by deepakraina99** [clariusdev/cast#158]
This script used to work seamlessly for me, but it is now giving me an error: "./pyclariuscast.so: undefined symbol: Caster" on this line of code: 
```cast = pyclariuscast.Caster(newProcessedImage, newRawImage, newSpectrumImage, newImuData, freezeFn, buttonsFn)
Please check.

**Comment by julien-l** [comment]
Ensure you have Cast lib version 11.2.0. To avoid compatibility errors, checkout the repository's tag version corresponding to your binaries version. Thanks.

**Comment by deepakraina99** [comment]
Yes, I am using the updated repository for 11.2.0. The latest binaries are only available for 11.1.0, so I am using them.

**Comment by julien-l** [comment]
Binaries version 11.2.0: https://github.com/clariusdev/cast/releases/tag/v11.2.0

If you want to use binaries version 11.1.0, ensure you checkout the respository tag `v11.1.0`, not `v11.2.0`.
